gnd/src/main/java/com/google/android/gnd/persistence/remote/firestore/schema/FeatureDocument.java

```diff
@@ -27,7 +28,12 @@
   @Nullable private String customId;
   @Nullable private String caption;
   @Nullable private GeoPoint location;
-  @Nullable private String geoJson;
+  /**
+   * Use geometry instead.
+   *
+   */
+  @Deprecated @Nullable private String geoJson;
```

Comment:
It may be premature to deprecate this. Sorry if I suggested this.

Reply:
fixed.